wire-ios/Wire-iOS/Sources/UserInterface/Availability/AvailabilityTitleViewController.swift

```diff
@@ -62,25 +64,22 @@ final class AvailabilityTitleViewController: UIViewController {
 
     func presentAvailabilityPicker() {
         let alertViewController = UIAlertController.availabilityPicker { [weak self] (availability) in
-            self?.didSelectAvailability(availability)
+            guard let `self` = self else { return }
+            self.didSelectAvailability(availability, userSession: self.userSession)
         }
 
         alertViewController.configPopover(pointToView: view)
 
         present(alertViewController, animated: true)
     }
 
-    private func didSelectAvailability(_ availability: AvailabilityKind) {
+    private func didSelectAvailability(_ availability: AvailabilityKind, userSession: UserSession) {
```

Comment:
Why inject this when you can get it from self?

Reply:
Fixed it!